2_embeddings.ipynb

```diff
@@ -16,29 +16,15 @@
   },
```

Comment:
<p><code>TensorBoardColab('runs/')</code>, don't need to assign it to a value I'm pretty sure, would be nice to call the dir <code>runs/</code> or something as that's more normal too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scoutbee/pytorch-nlp-notebooks/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
I got an error on colab when running this line, but without it TensorBoard still runs, so I commented out.